lib/Site.js

```diff
@@ -264,6 +274,10 @@ Site.prototype.createPage = function (config) {
                                path.join(this.siteAssetsDestPath, 'css', 'bootstrap.min.css')),
       bootstrapVue: path.relative(path.dirname(resultPath),
                                   path.join(this.siteAssetsDestPath, 'css', 'bootstrap-vue.min.css')),
+      fontAwesome: path.relative(path.dirname(resultPath),
+                                 path.join(this.siteAssetsDestPath, 'css', 'font-awesome.css')),
+      glyphicons: path.relative(path.dirname(resultPath),
+                                path.join(this.siteAssetsDestPath, 'css', 'glyphicons.css')),
```

Comment:
- Let's provide and use the minified versions.
- Let's name it **bootstrap-glyphicons.css** (and **.min.css**) to be explicit about its source.